<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-EBUL8M11" ignored="0" name="Test" tests="2" failures="1" timestamp="2021-01-06T12:18:19 COT" time="5.969" errors="0">
  <testcase name="correctLogin" time="4.529" classname="Pruebas.TestLogin"/>
  <testcase name="errorLogin" time="1.288" classname="Pruebas.TestLogin">
    <failure type="java.lang.AssertionError" message="expected [khkhkhkhkLogin failed] but found [Login failed]">
      <![CDATA[java.lang.AssertionError: expected [khkhkhkhkLogin failed] but found [Login failed]
at Pruebas.TestLogin.errorLogin(TestLogin.java:45)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- errorLogin -->
</testsuite> <!-- Test -->
